### PR TITLE
Add OmniTrade Exchange

### PR DESCRIPTION
**Pull request by adbatista**

### CODE REVIEW

**Comment by adbatista** [comment]
@wbnns done :)

File: _templates/exchanges.html [hunk 1]
```diff
@@ -63,6 +63,7 @@ <h3 id="brazil"><img src="/img/flags/BR.png" alt="Brazilian flag">Brazil</h3>
       <p>
         <a href="https://foxbit.com.br/">Foxbit</a><br>
         <a href="https://www.mercadobitcoin.com.br/">Mercado Bitcoin</a><br>
+        <a href="https://www.omnitrade.io/">OmniTrade</a>
```
Comment (wbnns) [inline _templates/exchanges.html]: A line break needs to be added at the end of this line, otherwise OmniTrade and Walltime will appear on the same line.

Comment (adbatista) [reply _templates/exchanges.html]: oh my Bad, forgot the `<br>`

now Is ok :)

**Comment by wbnns** [comment]
Unless others object, this will be merged on Wednesday, May 16th.

Will fix the merge conflict at that time.

**Pull request closed without merging**